Libraries/LibWeb/HTML/HTMLElement.cpp

```diff
@@ -775,7 +775,9 @@ Optional<ARIA::Role> HTMLElement::default_role() const
     // https://www.w3.org/TR/html-aria/#el-samp
     if (local_name() == TagNames::samp)
         return ARIA::Role::generic;
-    // https://www.w3.org/TR/html-aria/#el-section
```

Comment:
This comment appears to have been accidentally deleted.

Reply:
Fixed.